pkg/apis/storage/validation/validation.go

```diff
@@ -87,3 +94,20 @@ func validateParameters(params map[string]string, fldPath *field.Path) field.Err
 	}
 	return allErrs
 }
+
+var supportedReclaimPolicy = sets.NewString(string(api.PersistentVolumeReclaimDelete), string(api.PersistentVolumeReclaimRetain))
+
+// validateReclaimPolicy tests that the reclaim policy is one of the supported. It is up to the volume plugin to reject
+// provisioning for storage classes with impossible reclaim policies, e.g. EBS is not Recyclable
+func validateReclaimPolicy(reclaimPolicy *api.PersistentVolumeReclaimPolicy, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if reclaimPolicy == nil {
```

Comment:
we should not need to check nil here - defaulting should have set it.  Is there a reason?